docs/recipes/design-patterns/component-composition.md

```diff
@@ -0,0 +1,133 @@
+# Component composition/extending
+
+Since Skate is already close to the platform, you can easily extend other component classes in 2 ways:
+
+- via [Inheritance](#inheritance)
+- via [Mixins](#mixins)
+
+## Inheritance
+
+```js
+import { Component, h } from 'skatejs';
+
+// If you never use this class for an HTML element then you don't have to
+// ever register it as a custom element and it can still be extended.
+class BaseComponent extends Component {
+  static props = {
+    someBaseProp: {}
+  }
+}
+
+class SuperComponent extends BaseComponent {
+  renderCallback ({ someBaseProp }) {
+    return <div>{someBaseProp}</div>;
+  }
+}
+
+customElements.define('super-component', SuperComponent);
+```
+
+### Extending Props API via inheritance
+
+Let's say you want to inherit from some component which already defines some common `props`.
+
+**Solution is following:**
```

Comment:
Did you mean to say "the solution is as follows"? Maybe saying this isn't necessary as it's already implied?